src/VisualStudio/CSharp/Impl/LanguageService/CSharpCodeCleanupFixer.cs

```diff
@@ -309,6 +311,11 @@ private async Task<bool> FixTextBufferAsync(TextBufferCodeCleanUpScope textBuffe
                 cancellationToken = cancellationTokenSource.Token;
 
                 var buffer = textBufferScope.SubjectBuffer;
+
+                // Load document here so that the partial load message actually shows up before the next scope kicks in.
+                var document = buffer.CurrentSnapshot.GetFullyLoadedOpenDocumentInCurrentContextWithChangesAsync(
+                        context.OperationContext).WaitAndGetResult(cancellationToken);
```

Comment:
❗️ Do not call `WaitAndGetResult` from an asynchronous method. This should use `await` instead.